app/frontend/components/pages/accounts/accountDropdown.tsx

```diff
@@ -27,7 +20,7 @@ const AccountDropdown = ({
         onBlur={() => hideAccountDropdown(true)}
         onClick={() => toggleAccountDropdown()}
       >
-        {shouldNumberAccountsFromOne ? `Account #${accountIndex + 1}` : `Account ${accountIndex}`}
+        {`Account ${printAccountIndex(accountIndex)}`}
```

Comment:
for the sake of consistency, I'd name the function `formatAccountIndex`

Reply:
Sure, np, but with what exactly you mean it should be consistent with? We use the `printAda` helper in a very similar way so my initial idea was to be consistent with it.

Reply:
I think I was referring to `formatToken` which appeared in this PR as well

Reply:
You mean the one you suggested to be renamed to `parseToken` ? :D Anyway, I renamed it to format token. :+1:

Reply:
I see :D I think got confused by that formatToken token function and forgot to update this comment